ca/certificate-authority.go

```diff
@@ -65,6 +65,7 @@ type CertificateAuthorityImpl struct {
 	SA             core.StorageAuthority
 	PA             core.PolicyAuthority
 	Publisher      core.Publisher
+	KeyPolicy      core.KeyPolicy
```

Comment:
Could you make this field unexported and set it using an argument in the CA constructor? As you can see, we set some of the fields in CA and RA by making them exported and assigning to them after construction, but we've found that that's an anti-pattern and have been meaning to fix it: https://github.com/letsencrypt/boulder/wiki/Config-plan.